collagraph/renderers/pyside_renderer.py

```diff
@@ -222,16 +223,46 @@ def set_element_text(self, el: Any, value: str):
 
     def set_attribute(self, el: Any, attr: str, value: Any):
         """Set the attribute `attr` of the element `el` to the value `value`."""
+
+        key = f"{type(el).__name__}.{attr}"
+        if key not in DEFAULT_VALUES:
+            if not hasattr(el, "metaObject"):
+                logger.debug(f"{el} does not have metaObject")
+            else:
+                method_name = attr_name_to_method_name(attr, setter=False)
+
+                meta_object = el.metaObject()
+                property_idx = meta_object.indexOfProperty(method_name)
+                if property_idx >= 0:
+                    meta_property = meta_object.property(property_idx)
+                    result = meta_property.read(el)
+                    DEFAULT_VALUES[key] = (meta_property, result)
```

Comment:
are Meta Property objects from one QApp to another? I imagine this comes up in your tests?

Reply:
Well you're storing QMetaProperty objects in the global DEFAULT_VALUES.

During your test suite, the QApp is destroyed and a new one is created for the next test.

Is that a problem?